schemas/json/README.md

```diff
@@ -95,18 +104,17 @@ First, a [JSON number] can only be a concrete numerical value where special case
 Second, the [lexical space] of XSD values is much larger than the lexical space of JSON numbers and booleans.
 For example, both `1` and `true` are valid lexical representations of a true value.
 In contrast, JSON recognizes only `true`.
+We are not considering cases in which a semantic definition would even redefine the semantics of `1` as `false`.
 
 [lexical space]: https://www.w3.org/TR/xmlschema11-2/#lexical-space
 
-Since the semantic of the value is defined outside the model, and might depend directly on the lexical representation, we must represent these values as strings.
-For example, we might define through the attribute [Property/semanticId] that a [Property/value], given as a boolean, distinguishes between `1` and `true` where `1` has a completely different meaning from `true`.
-This distinction would have been lost if we simply serialized the value to a JSON boolean.
+In many cases the semantics of a [Property] or any other [SubmodelElement] is also restricting its value range in case of numerical values. However, when just looking at the [Property/valueType] this information is not given`and the value range seems larger than it actually is allowed to be.
```

Comment:
Unexpected `?

Reply:
@BirgitBoss please take a quick look at this line and the '`'.

Reply:
Sentence removed in the mean-time